### PR TITLE
Add system-information extension

### PR DESCRIPTION
**Pull request by Visual-Studio-Coder**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Visual-Studio-Coder** [comment]
@pernielsentikaer I just was wondering if there was a way I could have an action called "open system settings" and then open that about this mac thing, but I'm a bit unsure how to do it. Please give me some advice on doing this. Thanks in advance!

File: extensions/system-information/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      }
+
+      const serialNumberMatch = stdout.match(/Serial Number \(system\): (.+)/);
+      const serialNumber = serialNumberMatch ? serialNumberMatch[1] : null;
+
+      setSerialNumber(serialNumber);
+    });
+  }, []);
+
+  const releaseImage = () => {
```
Comment (pernielsentikaer) [inline extensions/system-information/src/index.jsx:44]: You forgot about Sonoma, and it would be cool if images were added to assets instead.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

I think you can do something like this (haven't tested)

```
actions={<ActionPanel><Action.Open target="x-apple.systempreferences:com.apple.Accessibility-Settings.extension" title="Open" /></ActionPanel>}
```

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Visual-Studio-Coder** [comment]
@pernielsentikaer I made some changes, and when I tried npm run publish, I got this
```
> publish
> npx @raycast/api@latest publish

ready -  validate git repository
ready -  getting fork
wait  -  preparing clone...failed running git command 'git checkout main', exit status 1

error: Your local changes to the following files would be overwritten by checkout:
        extensions/system-information/src/index.jsx
Please commit your changes or stash them before you switch branches.
Aborting
```

Please tell me how to fix this, but if not, then could you please just try to merge the repo I've already made (and updated) for me?

Here is the link to that repo: https://github.com/Visual-Studio-Coder/system-information

**Comment by pernielsentikaer** [comment]
Did you try to run `npx @raycast/api@latest pull-contributions`

**Comment by Visual-Studio-Coder** [comment]
Okay, some nice staff member helped me get this fixed, so shoutout to mathieudutour. I have changed the icon and added some more actions, like checking storage and updates.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**